Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/ForwardPass_StandardLighting_B.azsli

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli>
+#include <Atom/Features/Pipeline/Forward/ForwardPassSrg.azsli>
+
+// TODO(MaterialPipeline): I don't like how this file #includes something from BasePBR. I'd rather it include a file
+// called StandardPBR_LightingData.azsli which just #includes the BasePBR file. Otherwise this looks suspcious like
```

Comment:
s/suspcious/suspicious

FWIW, it looks suspicious perhaps because we are already familiar with what "Base" and "Standard" PBR mean in the context of our material system, but it's not unnatural for a more advanced lighting model to simply include and extend functionality from a simpler lighting model. I'll leave it to you if you think you'd rather keep this TODO though.

Reply:
At some point I expect to rename these files anyway because "StandardPBR" is the name of a material type, but these are just different lighting model code, so should be named like "StandardLightingModel". I'll address the TODO in some way at that point.